lib/glific_web/schema/users_types.ex

```diff
@@ -25,6 +25,11 @@ defmodule GlificWeb.Schema.UserTypes do
     end
   end
 
+  object :role do
+    field :id, :string
```

Comment:
I think the type of this should be id not string.

Reply:
We don't have ids for roles right now.
Update this response structure after discussing it with the frontend team.